docs/angular/guides/misc-data-persistence.md

```diff
@@ -192,3 +192,17 @@ class TodoEffects {
   constructor(private action$: Actions, private backend: Backend) {}
 }
 ```
+
+The StoreRouterConnectingModule must be configured with an appropriate serializer. The default serializer works.
```

Comment:
```suggestion
The StoreRouterConnectingModule must be configured with an appropriate serializer. The `DefaultRouterStateSerializer` provides the full router state instead of the `MinimalRouterStateSerializer` that is used without configuration.
```